app/pages/project/project-page.spec.js

```diff
@@ -2,27 +2,59 @@ import React from 'react';
 import { mount, shallow } from 'enzyme';
 import { expect } from 'chai';
 import { project, workflow } from '../dev-classifier/mock-data';
-import ProjectPage from './project-navbar';
+import ProjectPage from './project-page';
+
+function Page() {
+  return (
+    <p>Hello world!</p>
+  );
+}
 
 describe('ProjectPage', () => {
   const background = {
     src: 'the project background image url'
   };
 
   it('should render without crashing', () => {
-    shallow(<ProjectPage />);
+    shallow(<ProjectPage><Page /></ProjectPage>);
   });
 
   it('should render the project nav bar', () => {
-    const wrapper = shallow(<ProjectPage />);
+    const wrapper = shallow(<ProjectPage><Page /></ProjectPage>);
     const navElement = wrapper.find('ProjectNavbar').first();
     expect(navElement).to.have.lengthOf(1);
   });
 
   it('should receive a background, project and workflow props', () => {
-    const wrapper = mount(<ProjectPage background={background} project={project} workflow={workflow} />);
+    const wrapper = mount(
+      <ProjectPage background={background} project={project} workflow={workflow} >
+        <Page />
+      </ProjectPage>
+    );
     expect(wrapper.props().background).to.equal(background);
     expect(wrapper.props().project).to.equal(project);
     expect(wrapper.props().workflow).to.equal(workflow);
   });
+
+  describe('with a launch-approved project', () => {
+    project.launch_approved = true;
+    const wrapper = shallow(<ProjectPage project={project} ><Page /></ProjectPage>);
+    const disclaimer = wrapper.find('Translate[className="project-disclaimer"]');
+    it('should not render the Zooniverse disclaimer.', () => {
+      expect(disclaimer).to.have.lengthOf(0);
+    });
+  });
+
+  describe('without approval', () => {
+    project.launch_approved = false;
+    const wrapper = shallow(<ProjectPage project={project} ><Page /></ProjectPage>);
+    const disclaimer = wrapper.find('Translate[className="project-disclaimer"]');
+    it('should render the Zooniverse disclaimer.', () => {
+      expect(disclaimer).to.have.lengthOf(1);
+    });
+    it('should render the disclaimer immediately after its children.', () => {
+      expect(wrapper.childAt(1).name()).to.equal('Page');
+      expect(wrapper.childAt(2)).to.deep.equal(disclaimer);
```

Comment:
The test passes but, as we discussed, it's brittle and it'd break if we add anything to the page because the indexes are fixed. I think it's ok for now, but ideally we want to test that the _next_ child after `Page` is `disclaimer`.

Reply:
Yeah, checking the relative positions of the child page and the disclaimer in the components child nodes would be much better than checking the absolute positions of both.